11_webui.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+set -eux
+source common.sh
+
+figlet "Deploying web ui" | lolcat
+eval "$(go env)"
+
+
+export UIPATH="$GOPATH/src/github.com/kubevirt/web-ui-operator"
+
+oc new-project kubevirt-web-ui
+cd $UIPATH/web-ui-operator/deploy
+oc apply -f service_account.yaml
+oc adm policy add-scc-to-user anyuid -z kubevirt-web-ui-operator
+oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:kubevirt-web-ui:kubevirt-web-ui-operator
```

Comment:
Why is cluster-admin role needed?

Reply:
that wouldnt work without that when i tested it

Reply:
It is SA for the operator only. If roles are otherwise granted, this should not be required.
Do you have additional details, please?

Reply:
I did a bunch of re-deployment cycles today while enhancing the operator for HCO but I did not hit any issue requiring this role to be granted. Please try to remove it and if it fails for you, let me know - send logs, please.